net/nginx/files-luci-support/uwsgi.init

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh /etc/rc.common 
```

Comment:
uwsgi init script should be in that package.
you may need also two variants there.